README.md

```diff
@@ -43,7 +43,7 @@ To deploy the application:
    application](https://developers.google.com/appengine/docs/python/tools/uploadinganapp) with
 
    ```
-   appcfg.py -A <your-project-id> --oauth2 update .
+   appcfg.py -A <your-project-id> -V v1 update .
```

Comment:
Do you know the command to set the default version?

On Thu, Mar 3, 2016 at 1:57 PM Jerjou <notifications@github.com> wrote:

> In README.md
> <https://github.com/GoogleCloudPlatform/appengine-flask-skeleton/pull/22#discussion_r54955413>
> :
>
> > @@ -43,7 +43,7 @@ To deploy the application:
> >     application](https://developers.google.com/appengine/docs/python/tools/uploadinganapp) with
> >
> >     ```
> > -   appcfg.py -A <your-project-id> --oauth2 update .
> > +   appcfg.py -A <your-project-id> -V v1 update .
>
> Should we call that out in the text below? eg "if this is your first
> version deployed, you've got an app! otherwise, you gotta set the version
> to be default before you can see it, or go to this version-encoded url"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-flask-skeleton/pull/22/files#r54955413>
> .
>